modules/charon-core/src/main/java/org/wso2/charon3/core/schema/AttributeSchema.java

```diff
@@ -71,5 +72,9 @@
 
     public void removeSubAttribute(String subAttributeName) throws CharonException;
 
+    public List<String> getCanonicalValues();
+
+    public void setCanonicalValues(ArrayList<String> canonicalValues);
+
```

Comment:
Don't the methods in an interface always public? Do we need to declare them as public?